game/world/objects/npc.h

```diff
@@ -596,6 +596,9 @@ class Npc final {
     Npc*                           currentOther   =nullptr;
     Npc*                           currentVictim  =nullptr;
 
+    Npc*                           quickLookNpc=nullptr;
```

Comment:
Should it be mutually exclusive with `currentLookAtNpc` and occupy same variable?